Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -292,6 +292,24 @@ public static class Defaults
                 ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
                 Statistic = Statistic.Maximum,
                 Namespace = AwsNamespace.Rds
+            },
+            new AlarmDefinition
+            {
+                Name = "CPUSurplusCreditsChargedHigh",
+                Metric = "CPUSurplusCreditsCharged",
+                Enabled = false,
+                Period = TimeSpan.FromMinutes(5),
+                EvaluationPeriods = 1,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 0
+                },
+                DimensionNames = new[] { "DBInstanceIdentifier" },
+                ComparisonOperator = ComparisonOperator.GreaterThanThreshold,
+                Statistic = Statistic.Maximum,
+                Namespace = AwsNamespace.Rds,
+                TreatMissingData = TreatMissingDataConstants.Breaching
```

Comment:
Sorry, commenting on this again. I don't think Breaching is correct - surely if there's no data, nobody needs to be paged about excess charges, because there aren't any?

Reply:
Missing as you originally had it would probably be better

Reply:
Fair point. Shall I set it to `Ignore` then?

Reply:
Probably Missing is better, then you can see the alarm is in insufficient state in AWS console rather than "ok"

Reply:
Ok. Will update it.